### PR TITLE
Docker: Added Datadog support

### PR DESCRIPTION
**Pull request by rmanalan**
This PR adds support for the new Datadog integration as well as fixes some bugs I found when configuring the relay. The relay isn't required for this integration, just decided to fix it.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,38 +1,37 @@
...
     ports:
-      - "5000:5000"
-    command: ["./scripts/start_dev.sh", "5000"]
+      - 3000:3000
+    command: flask run -h 0.0.0.0 -p 3000 --reload
     depends_on:
-      - db
       - cache
 
-  db:
```
Comment (levlaz) [inline docker-compose.yml]: @rmanalan is there a specific reason why you got rid of the database here?

**Comment by levlaz** [comment]
@rmanalan I think there are some mistakes here -- for instance did you intend to remove this file? 

> scripts/start_dev.sh

File: docker-compose.yml (hunk 1 above)
Comment (rmanalan) [reply docker-compose.yml]: Because I realized that the dev env doesn't require PG and will default to sqlite.

**Comment by rmanalan** [comment]
> @rmanalan I think there are some mistakes here -- for instance did you intend to remove this file?
> 
> > scripts/start_dev.sh
Yeah, that was intentional since when I introduced this file, we needed to do a db migration and then run flask. All of that is done in the docker-compose.yml now.

**Comment by levlaz** [comment]
@rmanalan thanks for the comments, this LGTM.

File: requirements.txt [hunk 2]
```diff
@@ -24,3 +24,6 @@ launchdarkly-api==2.0.15
 
 # Runtime
 gunicorn==19.9.0
+
+# Datadog
+datadog
```
Comment (levlaz) [inline requirements.txt]: @rmanalan can you please pin this to a specific version?

Comment (rmanalan) [reply requirements.txt]: Done.

**Comment by levlaz** [comment]
Thanks @rmanalan this is good to go once https://github.com/launchdarkly/SupportService/pull/102 is merged.

File: .env.example [hunk 3]
```diff
@@ -19,3 +19,6 @@ LD_FRONTEND_KEY=
 # LD_RELAY_STREAM_URL=
 # Leave LD_API_KEY blank unless you need to connect to the API
 # LD_API_KEY=
+
+# Uncomment the following section to send metrics to Datadog. This is wired to the Support Service (Dev) account.
+# DD_API_KEY=83821f1a517bc8eb9a6573e7fede9568
```
Comment (InTheCloudDan) [inline .env.example]: @rmanalan we need to rotate this key, we can add secrets into the circleci config, this is a public repo

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]

File: .env.example (hunk 3 above)
Comment (rmanalan) [reply .env.example]: Ugh... sorry about that. I didn't realize this was public. Will rotate.

**Review by InTheCloudDan — APPROVED** [review]

**Comment by InTheCloudDan** [comment]
@levlaz is this good to go as far as you're concerned?

**Pull request merged**